test/LinalgTransform/double_tiling_alp.mlir

```diff
@@ -0,0 +1,33 @@
+// RUN: mlir-proto-opt -linalg-interp-transforms %s | FileCheck %s
+
+// CHECK-LABEL: func @matmul_tensors(
+// CHECK-SAME:    %[[TA:[0-9a-z]+]]: tensor<128x128xf32>
+// CHECK-SAME:    %[[TB:[0-9a-z]+]]: tensor<128x128xf32>
+// CHECK-SAME:    %[[TC:[0-9a-z]+]]: tensor<128x128xf32>
+// CHECK-SAME:  -> tensor<128x128xf32> {
```

Comment:
I'm not sure these check lines are necessary. The test seems to ensure the interpreter doesn't assert, so just having one label line and ensure some IR was produced is enough, the rest will just create churn should the syntax change in the future. Also consider adding a simple comment stating that only the success of the transformation is relevant here.